src/Providers/BBCodesProvider.php

```diff
@@ -11,12 +11,14 @@ class BBCodesProvider
 	public function __construct(Configurator $config)
     {
         $config->BBCodes->addCustom(
-            '[map provider={TEXT2} style={TEXT3} zoom={TEXT4}]{TEXT1}[/map]',
+            '[map provider={TEXT2} style={TEXT3} zoom={TEXT4} title={TEXT5?} desc={TEXT6?}]{TEXT1}[/map]',
             '<div class="bbcode-map"
               data-map-location="{TEXT1}"
               data-map-provider="{TEXT2}"
               data-map-style="{TEXT3}"
-              data-map-zoom="{TEXT4}">
+              data-map-zoom="{TEXT4}"
+              data-map-title="{TEXT5?}"
+              data-map-desc="{TEXT6?}">
```

Comment:
![image](https://user-images.githubusercontent.com/10224878/218129645-35ca36b8-47a4-4089-a0fa-f6ce0a6c0279.png)
The moment I activate those specific `TEXT5?` and `TEXT6?` I have got the entire site (front page included) that starts to raise that Exception, breaking all the Flarum instance.

Removing the `?` from `TEXT5?` and `TEXT6?` solves the problem.

Reply:
Removing the optional flas of `title` and `desc` would prevent older formatted `[map]` bbcode **not** rendering.
![image](https://user-images.githubusercontent.com/10224878/218132634-8435e28b-997e-48f6-9bc8-e4dd22dc5d2d.png)

Example above: 
  - first with `title` and `desc` does render.
  - the second without does not render.

That's a rather inconvenient regression.

Reply:
It sounds strange. It works well from my website: https://vietsuquan.com/d/1-chua-phu-xa-phuc-hoa-tu
I added 2 instances of bbcode map with and without popup content.

Reply:
Thanks for your feedback. Something might be wrong with my dev instance. I'll look into it and keep hou updated here.

Reply:
Somehow, in `BBCodesProvider.php` removing the `?` in the div part but not in the  [map]` solved the error I was having.